source/documentation/modules/color.liquid

```diff
@@ -0,0 +1,834 @@
+---
+title: sass:color
+---
+
+{% render 'documentation/snippets/built-in-module-status' %}
+
+{% function 'color.adjust($color,!!!  $red: null, $green: null, $blue: null,!!!  $hue: null, $saturation: null, $lightness: null,!!!  $whiteness: null, $blackness: null,!!!  $alpha: null)', 'adjust-color(...)', 'returns:color' %}
```

Comment:
We can't pass newlines as part of the signatures because 11ty crashes. Instead, I came up with this `!!!` idea to artificially insert newlines in the signature. See https://deploy-preview-54--sass-site-oddbird.netlify.app/documentation/modules/color/#adjust

Reply:
Clever! I tested locally and it looks like we can include newlines if we use [`{% capture %}`](https://liquidjs.com/tags/capture.html) to create the arg. I think that will be clearer, so I'll try that out.